yolov5/helpers.py

```diff
@@ -72,6 +72,7 @@ def video_predict(self, video_path, view_img=True, size=640, augment=False, line
         output = []
         while True:
             _, im = cap.read()
+            im = cv2.resize(im, (size, size))
```

Comment:
@kadirnar burada goruntuyu kareye mi ceviriyorsun, dogru mu anladim? yolov5 kare olmayan girdileri de destekliyor, niye boyle bir seye ihtiyac duyuyoruz?

Reply:
Load_images fonksiyonunda height ve width için güncelleme yapılmaktadır. Anladığım kadarıyla size değerine 32 ekleniyor. Bu yüzden önceki kod boyut hatası veriyordu. Bu durumu çözdüğüm durumda bile boxes değerlerinde kayma oluyordu. Bu yüzden scale_coords fonksiyonu da ekledim. Ana repoya daha sadık bir çalışma oldu.